docs/content/mobile/android/cloud-deploy.md

```diff
@@ -25,11 +25,13 @@ Altenatively, if you want to create your own authentication server, follow [this
 To do this:
   *  From the [SpatialOS Console](https://console.improbable.io/projects), select your deployment name to display the project **OVERVIEW** screen.
   * In the **OVERVIEW** screen, there’s a **Tag** field, add `dev_login` to the field.
-1. [Create a Development Authentication Token (SpatialOS documentation).](https://docs.improbable.io/reference/latest/shared/auth/development-authentication#developmentauthenticationtoken-maintenance)
-1. Create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and includes the functionality you want. In your Unity Editor, add this script it to your Android client-worker GameObject.
-1. The `MobileWorkerConnector` provides a `DevelopmentAuthToken` field. Still in your Unity Editor, make sure your Android client-worker GameObject is selected and in the Inspector, locate the script you just added to it. 
-1. In the Inspector, in the script’s drop-down window, there is a field to add the authentication token that you created. 
-1. In the same drop-down window, ensure that the checkbox `ShouldConnectLocally` is not checked.
+1. [Create a Development Authentication Token (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/auth/development-authentication#developmentauthenticationtoken-maintenance).
+
+    > While developing your game you may want your game clients to use the proper authentication flow without already having developed your own authentication and login servers (following the process described [here](https://docs.improbable.io/reference/13.6/shared/auth/integrate-authentication-platform-sdk)). To facilitate this, Improbable hosts both a development authentication service and a development login service for use in early-stage game development.
```

Comment:
this is not completely correct: 
1. the flow using your own server and this one are slightly different
2. this flow is necessary to be able to connect using a mobile device

Reply:
What's the "proper" authentication flow? Is that the flow here: https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication-platform-sdk? It's not clear.
Also, the link needs to be: 13.6 -> latest

Reply:
I think proper is the wrong word here. the whole wording is slightly off.

we have two flows:
the dev auth one which is the one we are using here and needs to be used for early-stage development
and the one that laura linked: which is the one  where you implement your own auth server

Reply:
This part is taken from the top of the 'Create a Development Authentication Token (SpatialOS documentation) page, which was linked in the previous step:
https://docs.improbable.io/reference/13.6/shared/auth/development-authentication

Reply:
if it was just taken from that part: should we just remove it here? They need to navigate to that link anyway

Reply:
Removed.